src/server_manager/electron_app/release_linux_action.sh

```diff
@@ -14,4 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-yarn do server_manager/electron_app/package_linux
+yarn do server_manager/electron_app/build
```

Comment:
We should update our documentation that we now need a SENTRY_DSN environment variable when building to get Sentry events

Reply:
See the README.